app/controllers/coronavirus_form/live_in_england_controller.rb

```diff
@@ -19,11 +20,14 @@ def submit
       respond_to do |format|
         format.html { render controller_path, status: :unprocessable_entity }
       end
-    elsif session[:live_in_england] == I18n.t("coronavirus_form.questions.live_in_england.options.option_no.label")
+    elsif @form_responses[:live_in_england] == I18n.t("coronavirus_form.questions.live_in_england.options.option_no.label")
+      session[:live_in_england] = @form_responses[:live_in_england]
       redirect_to not_eligible_england_url
     elsif session[:check_answers_seen]
+      session[:live_in_england] = @form_responses[:live_in_england]
       redirect_to check_your_answers_url
     else
+      session[:live_in_england] = @form_responses[:live_in_england]
```

Comment:
Seems odd to have this line repeated 3 times, but ok.

Reply:
Yeah, I struggled to find a way of only having this included once.  A separate `set_session_store` method seemed a bit of an overkill.